axum/src/extract/host.rs

```diff
@@ -0,0 +1,59 @@
+use super::{FromRequest, RequestParts};
+use async_trait::async_trait;
+use std::{convert::Infallible};
+
+/// Extractor that extracts the host from a request.
+#[derive(Debug, Clone, Default)]
```

Comment:
Not sure this should implement `Default`. Whats the use case?

```suggestion
#[derive(Debug, Clone)]
```

Reply:
Whoops, good catch, accident from copy-pasting query extractor as a starting point.